### DISCUSSION TITLE
Add API Function for Updating Panel Button Attributes and Labels

### DISCUSSION THREAD

**Discussion by at-codes** [GrapesJS/grapesjs#5021] [Ideas]
The only way I've seen to reliably update the panel views buttons attributes and label is to remove them and replace them with new buttons(new button id ie 'open-blocks-cusotm') with the same command id. This is clunky and seems to cause other issues as well. There appear to be other actions expecting the original button ids such as selecting the styles manager when a component is selected. 

Can we get some new functions for updating things like the title attributes and labels of the panel buttons?

**Comment by artf** [comment]
https://github.com/GrapesJS/grapesjs/issues/5020#issuecomment-1497266100
